src/game.cpp

```diff
@@ -7141,6 +7141,16 @@ void game::smash()
             return; // don't smash terrain if we've smashed a corpse
         }
     }
+    bool ter_or_furn = m.has_flag_ter_or_furn( "ELECTRIFIED", smashx, smashy );
+    if (ter_or_furn) {
+            body_part bp = random_body_part();
+            if(u.deal_damage( nullptr, bp, damage_instance( DT_ELECTRIC, rng( 5, 20 ) ) ).total_damage() > 0) {
+                add_msg(m_bad, _("You are shocked by the %1$s!"),
+                    ter_or_furn ? m.tername(smashx, smashy).c_str() : m.furnname(smashx, smashy).c_str());
```

Comment:
Well that's what I get for copy pasting. I'll fix it and the other one I saw in game.cpp as part of this PR I guess.